### ISSUE TITLE
DS Implementation - SkipNav

### ISSUE THREAD

**Issue by TylerAPfledderer** [ethereum/ethereum-org-website#12929]
_🚨As of the initial posting of this issue, a new design needs to be created in Figma by @nloureiro before executing this update. The issue can still be assigned by a contributor in the meantime._

Update the existing [`SkipNav` component](https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/SkipLink.tsx) to use a new design.

Currently, this component is fully custom in it's styles and handling of the link redirect to content in the given page. Recommend investigating the use of the [SkipNavLink and SkipNavContent](https://v2.chakra-ui.com/docs/components/skip-nav/usage) provided by Chakra to handle in consideration of accessibility.

Be aware that `SkipNavLink` under the hood renders a primitive `a` tag with needed logic passed to it. We should retain this render, as its only job is to send the user to the specified section of content. 

It also has it's own theme config `SkipLink`. [See source code](https://github.com/chakra-ui/chakra-ui/blob/v2/packages/components/src/skip-nav/skip-nav.tsx#L41-L49).

In summary of initial recommendations:
- Remove the custom `SkipLink` file in favor of keeping styling in the component theme configuration
- Any translated content and `href` can be supplied at the parent level `RootLayout`.
- Import `SkipNavLink` / `SkipNavContent` directly from Chakra

cc @pettinarip Do NOT assign me! 😅

**Comment by murluki** [comment]
Could you assign me, please?

**Comment by Dharmik79** [comment]
@wackerow I want to work on this issue, once the Figma design is updated.

**Comment by nloureiro** [comment]
@murluki are you able to do the Figma design proposal?

**Comment by nloureiro** [comment]
> @nloureiro yes, I can help you with this.

great! will you create a Figma file? or do you want me to create one.

**Comment by pettinarip** [comment]
@murluki just to clarify, are you doing the design and the implementation in the code? or just the design part? assigning to you in the meantime.

**Comment by nloureiro** [comment]
> @nloureiro could you please create the Figma file? @pettinarip I’ll be doing the front end coding part for this task.

go for it!!!
https://www.figma.com/design/EecunGD6oTh62h7rdukaFL/Skip-Nav-component?node-id=1%3A49&t=lHyaiG0J1dR9xp0Q-1

Thank you for your collaboration

**Comment by TylerAPfledderer** [comment]
@pettinarip @murluki I updated this issue's description to give better clarity in describing what Chakra's `SkipNavLink` renders.

**Comment by surajpsuresh** [comment]
Hi I would like to help out on this, is it possible to assign me also to this?

**Comment by surajpsuresh** [comment]
Okay cool

**Comment by surajpsuresh** [comment]
@murluki  Is it possible to talk to you regarding contributing, best practices and how to actually share design work on GitHub. 
I have been looking at tutorials online, but would rally appreciate talking to an actual person on how to navigate in GitHub.

Any time that is convenient for you would work!

Cheers!

**Comment by murluki** [comment]
@surajpsuresh of course, if you have telegram, hit me up at murluki_prg. I can share some tips with you :)

**Comment by surajpsuresh** [comment]
Done

**Comment by TylerAPfledderer** [comment]
@murluki and @surajpsuresh you may reach out to me if you need any clarification of the requirements! 😁

**Comment by surajpsuresh** [comment]
> @murluki and @surajpsuresh you may reach out to me if you need any clarification of the requirements! 😁

Sure @TylerAPfledderer will do, I guess @murluki has completed this for now, keeping an eye out for any new design tasks that may pop up.

**Comment by corwintines** [comment]
GM all!

I am trying to get caught up to speed on where we are at for this issue. 

@nloureiro I can't access that Figma file that you linked out to here. Is this still available?

I put this as blocked for now, as it sounds like design is needed here first.

**Comment by nloureiro** [comment]
> GM all!
> 
> I am trying to get caught up to speed on where we are at for this issue.
> 
> @nloureiro I can't access that Figma file that you linked out to here. Is this still available?
> 
> I put this as blocked for now, as it sounds like design is needed here first.


sorry about this, the new draft mode from fig broke the old links.... damn!!!

here is the link: 
https://www.figma.com/design/EecunGD6oTh62h7rdukaFL/Skip-Nav-component?node-id=1-49&t=G6bMA9l77bupwTjt-1

**Comment by minimalsm** [comment]
Hey @murluki, are you still looking to work on this?

**Comment by Baystef** [comment]
Hi @nloureiro @minimalsm , I'll love to work on this. Thanks. Please assign to me

**Comment by minimalsm** [comment]
Assigned @Baystef. Please note this should be implemented in Shadcn/Tailwind, not Chakra as the issue says. LMK if that's an issue, thanks again!

**Comment by Baystef** [comment]
> Assigned @Baystef. Please note this should be implemented in Shadcn/Tailwind, not Chakra as the issue says. LMK if that's an issue, thanks again!

It's not an Issue. Thanks
